billing_export_to_csv.php

```diff
@@ -41,6 +41,7 @@
 $f_cost = gpc_get_int( 'cost' );
 $f_from = gpc_get_string( 'from' );
 $f_to = gpc_get_string( 'to' );
+$f_subprojects = gpc_get_string( 'subprojects' );
```

Comment:
Boolean?

Reply:
Checkbox sends a string "on" by default and w3c says its value is a string. Should we tamper with its value in PHP side?